zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -588,6 +622,16 @@ impl<F: Field> BytecodeCircuitConfig<F> {
         let code_hash = challenges
             .evm_word()
             .map(|challenge| rlc::value(&bytecode.rows[0].code_hash.to_le_bytes(), challenge));
+        let code_hash_word = Word::new(
+            bytecode.rows[0]
+                .code_hash
+                .to_le_bytes()
+                .chunks(32 / 2)
+                .map(|bytes| Value::known(from_bytes::value(bytes)))
+                .collect_vec()
+                .try_into()
+                .unwrap(),
+        );
```

Comment:
I think we can use `Word::from(bytecode.rows[0].code_hash)` and then use `into_value`.

Reply:
Done